src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -496,9 +497,10 @@ const validateUserName = (
   }
 
   // check for duplicate names
-  const duplicateName =
-    new Set(users.map((user) => user.name)).size !== users.length;
-  if (duplicateName) {
+  const count = users.filter(
+    (user, index) => user.name === userName && index !== currentIndex
+  ).length;
+  if (count > 0) {
```

Comment:
Nice, I like this simplification very much! I was wondering, do we want to add the error even for the tab that had this username first? :thinking: